### DISCUSSION TITLE
Remove 2x xp weekends

### DISCUSSION THREAD

**Discussion by Kirishiki** [chromiecraft/chromiecraft#3552] [Suggestions]
Please remove the 2x weekends, so that everyone would level their char when they want, atm most people just slack around and wait for the 2x weekends and its effecting low lvl dungeons and battlegrounds generally!

**Comment by hofik19** [comment]
Do it other way around and add optional x2 exp rates available all time, not just weekends.

**Comment by ProhorovIlja** [comment]
@hofik19 is right, you know.
Keep x2 exp rates available all time for quests, kills, etc.
x2 is not x3 or x6.
It doesn't allow players to quickly level up, unless they just no life the game daily.
For someone, who can only play during weekends, holidays.
I won't be able to reach level 60, etc. in 2 days by doing RDFs.
Because it takes forever to find a group for RDFs from level 15 and higher.

**Reply by heyitsbench** [reply]
For the record, 2x quest XP has been made permanent already.
